tests/Jellyfin.Server.Implementations.Tests/Test Data/Updates/manifest.json

```diff
@@ -540,7 +540,7 @@
     {
         "guid": "022a3003-993f-45f1-8565-87d12af2e12a",
         "name": "InfuseSync",
-        "description": "This plugin will track all media changes while any Infuse clients are offline to decrease sync times when logging back in to your server.",
+        "description": "This plugin will track all media changes while any Infuse clients are offline to decrease sync times when logging back into your server.",
```

Comment:
This change tends to be controversial. -- I'm happy to drop any particular changes.

Some people argue that `log in` is a specific thing. I personally believe that `log` is the verb and that this `into` isn't special.